provider/pkg/gen/nodejs-templates/yaml/yaml.tmpl

```diff
@@ -399,8 +399,9 @@ export class ConfigFile extends CollectionComponentResource {
     // for resource definitions that can be managed by Kubernetes, and registers those with the
     // engine instead.
     if (
-{{range $idx, $v := .ListKinds -}} {{if $idx}}
-        || {{else}}           {{end}}(gvk === "{{$v.GVK}}")
+           (gvk === "v1/List")
```

Comment:
Has this always been hand-coded, or was it auto-generated prior to the switch to schema codegen?

Reply:
`v1/List` was hardcoded before.